src/apps/context.ts

```diff
@@ -0,0 +1,22 @@
+/* eslint-disable @typescript-eslint/no-empty-function */
+import React from "react";
+
+export interface AppListContextValues {
+  activateApp: (appId: string) => void;
+  deactivateApp: (appId: string) => void;
+}
+
+export const AppListContext = React.createContext<AppListContextValues>({
+  activateApp: () => {},
+  deactivateApp: () => {}
+});
```

Comment:
The `!context` check you're doing inside `useAppListContext` won't work unless you omit the values on context instantiation. 

If you simply omit it here, it will work as expected without any other changes (and you can remove the eslint-disable comment as well!)
```suggestion
export const AppListContext = React.createContext<AppListContextValues | undefined>();
```

Reply:
Thanks, I forgot to delete the default data 😅